src/webrequest.js

```diff
@@ -55,6 +55,10 @@ function onBeforeRequest(details){
     recordFrame(details.tabId, details.frameId, details.parentFrameId, details.url);
   }
 
+  if (type == "ping" && details.tabId < 0 ){
```

Comment:
If we want to blanket-block all pings, yes, we should remove the `tabID` check.

Very soon, we'll have access to tabId for the pings from visible tabs (#522124).
The `if` clause will make sure we treat pings as any other HTTP request (i.e. not blanket-block but check the origin, dest, blacklist etc.)

Let me know which one you'd prefer.